internal/flypg/node.go

```diff
@@ -225,14 +231,6 @@ func (n *Node) Init(ctx context.Context) error {
 
 // PostInit are operations that need to be executed against a running Postgres on boot.
 func (n *Node) PostInit(ctx context.Context) error {
-	if ZombieLockExists() {
```

Comment:
Instead of returning, allow it to be re-screened there are certain failure conditions that will be remediated automatically.  If not, it will enter a crash loop and until the machines max restarts are hit.